modules/restore-regenerate-etcd-certs.adoc

```diff
@@ -0,0 +1,173 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-regenerate-etcd-certs_{context}"]
+= Regenerating etcd certificates
+
+If the etcd certificates have expired for your master host, you must regenerate them before you can add the member back to the etcd cluster.
```

Comment:
If the node is a new node or the etcd certificates on the node are no longer valid, you must regenerate them before you can add the member back to the etcd cluster.